### DISCUSSION TITLE
Erreur dans la création du projet

### DISCUSSION THREAD

**Discussion by meiry11** [AdrienDSlone/slowTripPlanner#43] [5. Back]
Bonjour à tous,
l'erreur que nous avions rencontres pendant la création du projet provenait de sa configuration avec intelij.
Je me suis souvenu que j'avais eu le même problème. Il ne reconnait pas le projet comme étant un projet Maven donc il faut faire un clic droit sur le projet rajouter le Framework Maven.
On verra le src apparaitre mais sans la class main par défaut de Springs donc évidemment d'autres configurations à faire.
J'ai donc créer un nouveau projet en local , nous verrons ensemble le lundi si on le garde ou pas.
Pour ce weekend je vais commencer la création des class et si possible le mapping.
Quant à la maquette pour la page home je le ferai dans la semaine vu que nous avons besoin de nos classe pour pouvoir avancer avec Romain. Je souhaite à tous de passer un très bon weekend.

**Comment by AdrienDSlone** [comment]
Bonjour Meiry,
Bien vu !
Entendu, je pense que si tu as déjà les classes de fonctionnelles tu peux même push sur une branche dédiée et on mergera sur la branche develop après échange.
Bonne fin de week-end !
